exporter/signalfxexporter/factory_test.go

```diff
@@ -231,13 +231,20 @@ func TestDefaultTranslationRules(t *testing.T) {
 
 	// disk_ops.total gauge from system.disk.operations cumulative, where is disk_ops.total
 	// is the cumulative across devices and directions.
+	// Test uses 4 metrics, each with more than 1 data point. Aggregate by sum reduces each metric to 1 data point.
+	// 4 metrics, each with one data point, create 3 delta data points, either how much the current data point is
+	// larger than previous, or current data point's value (if smaller than previous).
```

Comment:
I don't think this is an acceptable solution. We need to make sure that users see the same values in the charts built with `disk_ops.total` metric. I don't think it's going to be true if we break down `disk_ops.total` into several delta metrics. Please correct me if I'm wrong. 

If it indeed introduces different results, I see the following options to restore the same `disk_ops.total` metric:
1. Introduce a `metricstransform` processor in the default Splunk Otel distribution config to get a metric combined from `system.disk.operations.read` and `system.disk.operations.write`.
2. Refactor the signalfx exporter translation logic to work on a metrics batch instead of just one metric.

1 approach seems simpler and more reliable to me.